src/app/auth/login/login.component.ts

```diff
@@ -9,23 +10,33 @@ import { UserService } from '@shared/user.service';
 })
 export class LoginComponent implements OnInit {
   loginForm: FormGroup;
+  errorMessage = '';
 
-  constructor(private userService: UserService) {
+  constructor(private userService: UserService, private router: Router) {
   }
 
   ngOnInit() {
     this.loginForm = new FormGroup({
       login: new FormControl('', {
-        validators: [Validators.required, Validators.email]
+        validators: [Validators.required]
       }),
-      password: new FormControl('', { validators: [Validators.required] })
+      password: new FormControl('', {
+        validators: [Validators.required]
+      })
     });
-  }
+    }
 
   onSubmit() {
     this.userService.login({
       name: this.loginForm.value.login,
       password: this.loginForm.value.password
-    });
+    })
+      .subscribe((response) => {
```

Comment:
need to be unsubscribed, use Subscription or takeUntil operator

Reply:
Unused param `response`

Reply:
But it is an http request. Should I unsubscribe from it anyway?

Unused param removed.